apps/lnbits/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  web:
+    image: louneskmt/lnbits:v0.2.0
+    init: true
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - "$APP_LNBITS_PORT:$APP_LNBITS_PORT"
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+      - ${LND_DATA_DIR}:/lnd:ro
+    environment:
+      # Global
+      PORT: $APP_LNBITS_PORT
+      LNBITS_DATA_FOLDER: "/data"
+
+      # LND
+      LNBITS_BACKEND_WALLET_CLASS: LndWallet
+      LND_GRPC_ENDPOINT: $LND_IP
+      LND_GRPC_PORT: $LND_GRPC_PORT
+      LND_GRPC_CERT: "/lnd/tls.cert"
+      LND_GRPC_MACAROON: "/lnd/data/chain/bitcoin/$BITCOIN_NETWORK/admin.macaroon"
+
+      # App
+      LNBITS_SITE_TITLE: "LNbits - Umbrel"
+      LNBITS_DEFAULT_WALLET_NAME: "LNbits wallet"
+      LNBITS_DISABLED_EXTENSIONS: "amilk"
```

Comment:
Why is this extension disabled?

Reply:
It allows an user to automatically claim sats from an LNURLw. For example, a LNURL to claim 1000 sats every minute. I found this pretty unfair as the delay is here to allow more people to claim sats (e.g. a test LNURL in a Bitcoin bar, for newbies to discover Lightning) and not just have a single person claiming the whole balance. 

If necessary I can enable it.

Reply:
Hmmn, I agree that doesn't sound like very nice behaviour but I'm not sure it should be our decision to blacklist it.

It's not doing anything bad by default, right? A user would have to enable manually enable it?

Reply:
> A user would have to manually enable it?

Each extension has to be activated from the extensions page.

Reply:
But of the extension is disabled in this env var, it cannot be installed from the web page. So yeah, if it shouldn't be our decision to blacklist it, we should remove this line.

Reply:
Keep it disabled until we manage to improve it for future LNbits release